USAGE.md

```diff
@@ -177,6 +179,49 @@ Faker.Date.date_of_birth(10..19) #=> ~D[2004-05-15]
 Faker.Date.forward(4) #=> ~D[2016-12-25]
 
 Faker.Date.backward(4) #=> ~D[2016-12-20]
+
+Faker.Date.between(~D[2016-12-20], ~D[2016-12-25]) #=> ~D[2016-12-23]
+```
+
+### Faker.DateTime
+
+```elixir
+Faker.DateTime.forward(4)
+#=> %DateTime{calendar: Calendar.ISO, day: 25, hour: 6,
+#=>  microsecond: {922180, 6},  minute: 2, month: 12, second: 17,
+#=>  std_offset: 0, time_zone: "Etc/UTC", utc_offset: 0, year: 2016,
+#=>  zone_abbr: "UTC"}
+
+Faker.DateTime.backward(4)
+#=> %DateTime{calendar: Calendar.ISO, day: 20, hour: 6,
+#=>  microsecond: {922180, 6},  minute: 2, month: 12, second: 17,
+#=>  std_offset: 0, time_zone: "Etc/UTC", utc_offset: 0, year: 2016,
+#=>  zone_abbr: "UTC"}
+
+Faker.DateTime.between(from_datetime, to_datetime)
```

Comment:
I was wrong about `NaiveDateTime.t`. Updated `USAGE.md`. Let me know what you think.